### ISSUE TITLE
ResendEmailConfirmation has bad link

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/aspnetcore#24273]
1. Create a new ASP.NET Core 3.1 app with individual authentication.
2. Scaffold Login and Register pages.
3. The resend email link in the login page redirects to `https://localhost:5001/?page=%2FAccount%2FResendEmailConfirmation`, and it just opens the home page.
4. Scaffold the ResendEmailConfirmation.
5. The link now is correct and redirects to `https://localhost:44337/Identity/Account/ResendEmailConfirmation` which generates an error because of extra `abstract ` keyword `InvalidOperationException: Can't compile a NewExpression with a constructor declared on an abstract class`. (This was already solved before, but the fix seems to be not shipped with .NET Core 3.1).

To summarize, there are 2 problems:

1. Wrong link before scaffolding `ResendEmailConfirmation`
2. Correct link after scaffolding `ResendEmailConfirmation` but with the extra abstract keyword (which is fixed but not shipped with .NET Core 3.1).

The main problem for me is the first, I don't want to override anything in `ResendEmailConfirmation`, but I've to scaffold it.

I'm not sure if the two problems are related to each others or not.

**Comment by HaoK** [comment]
This file doesn't exist in 3.1 as we removed it to bring the feature back in 5.0, so I don't think we are likely going to be fixing it for 3.1, but the feature should work in 5.0 https://github.com/dotnet/aspnetcore/tree/release/3.1/src/Identity/UI/src/Areas/Identity/Pages/V4/Account (note there is no ResendEmailConfirmation.cshtml file)

**Comment by Youssef1313** [comment]
@HaoK, The 3.1 branch in Scaffolding repo contains the ResendEmailConfirmation file, should it be removed, and also remove the link for it in login page?

I opened https://github.com/dotnet/Scaffolding/pull/1375 a days ago to remove the abstract keyword from Scaffolding repo, but as far as I understood from you now, the change in that PR is incorrect, and the file should completely be deleted, is that correct?

**Comment by HaoK** [comment]
Correct, 3.1 should just remove the file and the link, it should be in 5.0 only, thanks for opening these PRs @Youssef1313

**Comment by Youssef1313** [comment]
@HaoK, Thanks. I've updated https://github.com/dotnet/Scaffolding/pull/1375. That PR should fix this issue.
